### DISCUSSION TITLE
It would be nice to have the branches in the sidebar

### DISCUSSION THREAD

**Discussion by tscholiadis** [Murmele/Gittyup#361] [Ideas]
I need to switch between branches a lot more often that I need to switch between projects. It would be really useful of I could add (or change) the left sidebar to contain a list of all the projects of the open project.

**Comment by Murmele** [comment]
Hi @tscholiadis,
do you know the branch combobox above the commit list?
![grafik](https://user-images.githubusercontent.com/10099533/211308570-67c15977-b8c0-44bb-a4da-87c63b56986d.png)

There you can change easily. Can you write an issue to track your request?

**Reply by tscholiadis** [reply]
Hi @Murmele and thank you for your reply.

Yes, I am aware of the drop down, but that's not very convenient when you're changing often. I'm referring to something like this:
![image](https://user-images.githubusercontent.com/994212/211559677-49cc5ce1-526e-41b4-a47c-3cfa3f214f07.png)

**Reply by Murmele** [reply]
Yes I understand. Just write a feature request in the issues, also with this image so somebody can implement it
